app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -61,54 +61,8 @@ export const TabBar = () => (
   </svg>
 );
 
-const Twitter = ({ tweets }) => {
-  const previous = takeLast(1, splitAt(1, tweets))[0];
-  return (
-    <PanelWrapper>
-      <Tweet tweet={tweets[0]} isPrimary />
-      <PreviousWrapper>
-        <TweetDivider>Previous Tweets</TweetDivider>
-        <TabBar />
-        {previous.map(tweet => (
-          <React.Fragment key={tweet.createdAt}>
-            <Tweet tweet={tweet} isPrimary={false} />
-            <TabBar />
-          </React.Fragment>
-        ))}
-      </PreviousWrapper>
-    </PanelWrapper>
-  );
-};
-
-Twitter.propTypes = {
-  tweets: PropTypes.arrayOf(
-    PropTypes.shape({
-      createdAt: PropTypes.string.isRequired,
-      text: PropTypes.string.isRequired,
-      interactions: PropTypes.shape({
-        favoriteCount: PropTypes.number.isRequired,
-        retweetCount: PropTypes.number.isRequired,
-      }).isRequired,
-      media: PropTypes.shape({
-        images: PropTypes.arrayOf(
-          PropTypes.shape({
-            id: PropTypes.number.isRequired,
-            mediaUrl: PropTypes.string.isRequired,
-          }),
-        ),
-        link: PropTypes.string,
-      }),
-      user: PropTypes.shape({
-        name: PropTypes.string.isRequired,
-        handle: PropTypes.string.isRequired,
-        avatar: PropTypes.string.isRequired,
-      }).isRequired,
-    }),
-  ).isRequired,
-};
```

Comment:
This looks like a refactor in the middle of a feature.
* Why are we consolidating these two components?
* If we need to consolidate the components, we should split them into separate commits.

Reply:
I've removed all aspects of the refactor now besides references to startTimer, and renaming of the default component - it's in a separate commit from the other changes.

Reply:
If you're undoing changes you no longer, they should be squashed in to the originating commit. Otherwise the edit/undo is just noise.

If you're renaming the default component, that's a refactor. And it should precede your feature commit as a separate unit of work.